databuilder/README.md

```diff
@@ -1678,6 +1678,9 @@ Adds the same set of tags to all tables produced by the job.
 #### [GenericTransformer](./databuilder/transformer/generic_transformer.py)
 Transforms dictionary based on callback function that user provides.
 
+#### [ComplexTypeTransformer](./databuilder/transformer/complex_type_transformer.py)
```

Comment:
let's remove this for now and then document everything once there's end to end support